core-plugins/src/test/java/co/cask/hydrator/plugin/batch/source/FileBatchSourceTest.java

```diff
@@ -569,45 +510,72 @@ public void testFileBatchInputFormatAvro() throws Exception {
 
     DatumWriter<GenericRecord> datumWriter = new GenericDatumWriter<>(avroSchema);
     DataFileWriter<GenericRecord> dataFileWriter = new DataFileWriter<>(datumWriter);
-    dataFileWriter.create(avroSchema, fs.create(avroFilePath));
+    dataFileWriter.create(avroSchema, fileAvro);
     dataFileWriter.append(record);
     dataFileWriter.close();
     inputManager.flush();
 
-    WorkflowManager workflowManager = appManager.getWorkflowManager(SmartWorkflow.NAME);
-    workflowManager.start();
-    workflowManager.waitForRuns(ProgramRunStatus.COMPLETED, 1, 2, TimeUnit.MINUTES);
+    workflowStartAndWait(appManager);
+
+    List<StructuredRecord> input = ImmutableList.of(
+      StructuredRecord.builder(RECORD_SCHEMA)
```

Comment:
this is the wrong output. The pipeline is deployed with 'pathField' as 'file', which means the output is supposed to contain a 'file' field whose value is the URI of the file.